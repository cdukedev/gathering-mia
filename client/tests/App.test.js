// import React from "react";
// import App from "../src/App";
// import renderer from "react-test-renderer";

// it("renders App component correctly", () => {
//   const tree = renderer.create(<App />).toJSON();
//   expect(tree).toMatchSnapshot();
// });
