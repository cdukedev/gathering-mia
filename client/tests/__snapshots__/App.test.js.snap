// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App component correctly 1`] = `
<div
  className="desktop"
>
  <div
    className="desktop-logo__container"
  >
    <img
      alt="logo"
      className="desktop-logo"
      src=""
    />
  </div>
  <div
    className="desktop-content__container"
  >
    <p
      className="desktop-content"
    >
      We are excited to have you join us! This site is intended for mobile use. Please scan the QR code to navigate to our site on your mobile device.
    </p>
    <div>
      <img
        alt="gathering website QR Code"
        className="desktop-qr"
        src=""
      />
    </div>
    <span
      className="desktop-content"
    >
      You may also type gathering-mia.live into your favorite mobile browser.
    </span>
  </div>
</div>
`;
